drivers/scheduler/k8s/k8s.go

```
@@ -1104,10 +1100,8 @@ func (k *K8s) Destroy(ctx *scheduler.Context, opts map[string]bool) error {
 			}
 			return nil, false, nil
 		}
-		pods, err = task.DoRetryWithTimeout(t, k8sDestroyTimeout, DefaultRetryInterval)
-		if err != nil {
-			podList = append(podList, pods.(v1.Pod))
-		}
+
+		task.DoRetryWithTimeout(t, k8sDestroyTimeout, DefaultRetryInterval)
```

we cannot ignore the err here.

---

same for other places in PR.